config/resources/organisation.lisp

```diff
@@ -1,11 +1,42 @@
+(define-resource organization ()
+  :class (s-prefix "org:Organization")
+  :properties `((:name :string ,(s-prefix "skos:prefLabel")))
```

Comment:
```suggestion
  :properties `((:naam :string ,(s-prefix "skos:prefLabel")))
```
Perhaps we should try to be consistent with bestuurseenheid to avoid confusion in the future? see https://github.com/lblod/app-subsidiepunt/blob/0dae1b19fcb8011cbd9c995fbba3a6b6cdff22d2/config/resources/organisation.lisp#L10